src/NexusMods.App.UI/Pages/LibraryPage/LibraryItemModel.cs

```diff
@@ -118,7 +120,8 @@ public static IColumn<LibraryItemModel> CreateNameColumn()
                 IsTextSearchEnabled = true,
                 CanUserResizeColumn = true,
                 CanUserSortColumn = true,
-            }
+            },
+            width: GridLength.Auto
```

Comment:
Default is `Auto`, so we don't need this.

Reply:
Yeah, I was experimenting with star, which had the usual issue of column collapsing on itself. I'm going to investigate treeDataGrid sizing this weekend, there is definitely a bug there to fix